chromatic_fitting.py

```diff
@@ -310,8 +452,9 @@ def set_name(self, name):
         self.name = name
 
     def get_prior(self, i, *args, **kwargs):
+        data = self.get_data()
         self.degree = self.parameters["p"].inputs['shape'] - 1
-        x = self.data.time.to_value()
+        x = data.time.to_value()
```

Comment:
Unless we can pass astropy units through the PyMC3 models (can we?), you should probably make this `data.time.to_value('days')` to be confident about the units you're getting the values in. Otherwise, a small change to the default units in the Rainbow could really change how we interpret the model.

Reply:
Good point!